### PR TITLE
update consensus information

### LINKED ISSUE #198: Make "proof of X" usage consistent
## Describe the bug

"Proof of X" usage inconsistent throughout docs.

- [ ] Missing content
- [ ] Outdated content
- [ ] Wrong content
- [x] Confusing or misleading content
- [ ] Other

## Change suggestion

Make usage of the consensus types "proof of X" (e.g., proof of authority, proof of work) consistent across the docs. Make them all lowercase and without hyphens: "proof of authority," not "Proof of Authority" or "proof-of-authority."

### PR DESCRIPTION
**Pull request by alexandratran**
# Pull request checklist

Use the following template to make sure your PR fits the GoQuorum documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-locally).

## Describe the change

Update consensus info, including:

- Add clear recommendation of QBFT to appropriate places, and clarify what's supported.
- Reorder pages to prioritize QBFT.
- Minor edits across consensus pages.
- Add migration info for Clique.

## Issue fixed

fixes https://github.com/ConsenSys/protocol-misc/issues/576
fixes #198 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools (e.g., lint)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [x] You've [previewed your changes on GitHub Pages](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-on-github-pages)

### CODE REVIEW

File: docs/concepts/consensus/comparing-poa.md [hunk 1]
```diff
@@ -68,11 +70,11 @@ Networks with three or fewer validators can produce blocks but do not guarantee
 
 !!! important
 
-    We do not recommend using IBFT and QBFT networks with three nodes for production purposes.
+    We don't recommend using QBFT and IBFT networks with three nodes for production purposes.
```
Comment (bgravenorst) [inline docs/concepts/consensus/comparing-poa.md:73]: Nit: It's an existing issue, but would it be better to say, "We recommend using QBFT and IBFT with at least four nodes in production environments."

File: docs/concepts/consensus/overview.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # Consensus protocols
 
-GoQuorum implements the following Proof of Authority consensus protocols:
+GoQuorum implements the following proof of authority (PoA) consensus protocols:
 
-* [IBFT](../../configure-and-manage/configure/consensus-protocols/ibft.md)
-* [QBFT](../../configure-and-manage/configure/consensus-protocols/qbft.md)
-* [Raft](../../configure-and-manage/configure/consensus-protocols/raft.md)
-* [Clique](../../configure-and-manage/configure/consensus-protocols/clique.md).
+* [QBFT](../../configure-and-manage/configure/consensus-protocols/qbft.md): QBFT is the recommended enterprise-grade
```
Comment (bgravenorst) [inline docs/concepts/consensus/overview.md:5]: Probably don't need to repeat the consensus protocol in these descriptions. For example, we can just say, "QBFT: the recommended enterprise-grade..."

File: docs/configure-and-manage/configure/consensus-protocols/ibft.md [hunk 3]
```diff
@@ -4,7 +4,12 @@ description: Configuring IBFT consensus
 
 # Configuring IBFT consensus
 
-GoQuorum implements the [IBFT](https://medium.com/getamis/istanbul-bft-ibft-c2758b7fe6ff) Proof-of-Authority (PoA) consensus protocol.
+GoQuorum implements the [IBFT](https://medium.com/getamis/istanbul-bft-ibft-c2758b7fe6ff) proof of authority
+[consensus protocol](../../../concepts/consensus/overview.md).
+IBFT is supported for existing users for private networks, but [QBFT](qbft.md) is the recommended enterprise-grade
+consensus protocol for private networks.
+You can [migrate an existing network using IBFT 2.0 to QBFT](qbft.md#migrate-from-ibft-to-qbft).
```
Comment (bgravenorst) [inline docs/configure-and-manage/configure/consensus-protocols/ibft.md:11]:
```suggestion
You can [migrate an IBFT network to QBFT](qbft.md#migrate-from-ibft-to-qbft).
```

File: docs/configure-and-manage/configure/consensus-protocols/ibft.md [hunk 4]
```diff
@@ -4,7 +4,12 @@ description: Configuring IBFT consensus
 
 # Configuring IBFT consensus
 
-GoQuorum implements the [IBFT](https://medium.com/getamis/istanbul-bft-ibft-c2758b7fe6ff) Proof-of-Authority (PoA) consensus protocol.
+GoQuorum implements the [IBFT](https://medium.com/getamis/istanbul-bft-ibft-c2758b7fe6ff) proof of authority
+[consensus protocol](../../../concepts/consensus/overview.md).
+IBFT is supported for existing users for private networks, but [QBFT](qbft.md) is the recommended enterprise-grade
```
Comment (bgravenorst) [inline docs/configure-and-manage/configure/consensus-protocols/ibft.md:9]:
```suggestion
IBFT is supported for existing private networks, but [QBFT](qbft.md) is the recommended enterprise-grade
```

**Review by bgravenorst — COMMENTED** [review]
Left some feedback

**Review by rolandtyler — APPROVED** [review]
lgtm

**Pull request merged**